packages/flutter/lib/src/services/text_input.dart

```diff
@@ -1441,7 +1467,7 @@ TextInputAction _toTextInputAction(String action) {
       return TextInputAction.next;
     case 'TextInputAction.previous':
       return TextInputAction.previous;
-    case 'TextInputAction.continue_action':
+    case 'TextInputAction.continueAction':
```

Comment:
Was this not working?

Reply:
Yes. There was an exception saying that `TextInputAction.continueAction`, after being converted to a string (sent by the platform channel) can not be converted back to an `TextInputAction`. This error has always been there, but was ignored because the platform messengers swallow errors. This PR fixes this problem and discovered this issue.